Gemfile

```diff
@@ -1,3 +1,3 @@
 gemspec
 
-gem 'aead', github: 'onelogin/aead', ref: 'master'
+gem 'aead', github: 'shopify/aead', ref: 'master'
```

Comment:
I know we're protected from errant upgrades by the `Gemfile.lock`, but can we specify the specific SHA here for extra-super-special explicitness (as we do in our internal `Gemfile`s when referencing the ActiveMerchant repo)?